src/registrar/views/domain.py

```diff
@@ -791,10 +791,11 @@ class DomainInvitationDeleteView(DomainInvitationPermissionDeleteView, SuccessMe
     object: DomainInvitation  # workaround for type mismatch in DeleteView
 
     def get_success_url(self):
+        messages.success(self.request, self.get_success_message())
```

Comment:
**question (non-blocking)**: The `SuccessMessageMixin` should take care of putting the success message on this page. Why did we need to add this?

Reply:
As described here: https://github.com/cisagov/manage.get.gov/pull/1767#discussion_r1486862871